### DISCUSSION TITLE
Navbar dropdown v.5.2.0

### DISCUSSION THREAD

**Discussion by ed3** [twbs/bootstrap#36784] [General] [answered]
In boostrap v.5.2.0 require Popper to run navbar dropdown, is correct ?
I run boostrap.js v.5.2.0 and have in console an error to line 2209. if I run boostrap.bundle.js v.5.2.0 working correctly. (bootstrap.js v.5.1.3 run correctly the navbar dropdown without bundle)

**Comment by TravelTimN** [comment]
Popper is included as part of the bundle for 5.2.x - so that's why it'll work.
If you're not using the bundle, then the following will not work, because Popper isn't included (dropdowns, poppers, and tooltips).
Make sure to either include Popper separately, if you're not using the bundle.

**Comment by GeoSot** [comment, marked as answer]
Hello @ed3 

You are correct. As dropdown in general was using the Popper engine, the check for navbar was moved to an inner depth, merging some functionalities, and on 5.3 Popper is going to be used on any case  #35901

**Comment by ed3** [comment]
Thank you @GeoSot and @TravelTimN !
Now it makes sense for Navbar Dropdown from version 5.2.0 requires Popper.
